### PR TITLE
Change default `CENTOS_VERSION` to 8

### PR DESCRIPTION
**Pull request by anoopcs9**

### CODE REVIEW

File: jobs/samba-integration.yml [hunk 1]
```diff
@@ -56,7 +56,7 @@
     - shell: !include-raw: scripts/common/get-node.sh
       # the samba-integration:centos-ci branch contains a copy of bootstrap.sh
       # from the gluster/centosci repo
-    - shell: ./bootstrap.sh samba-integration-centos-ci-tests.sh "ghprbPullId=${ghprbPullId} ghprbTargetBranch=${ghprbTargetBranch}"
+    - shell: ./bootstrap.sh samba-integration-centos-ci-tests.sh "ghprbPullId=${ghprbPullId} ghprbTargetBranch=${ghprbTargetBranch} CENTOS_VERSION=$CENTOS_VERSION"
```
Comment (obnoxxx) [inline jobs/samba-integration.yml:59]: If I understand this correctly, this hunk is not actually about changing the default, it is about handing the variable `CENTOS_VERSION` into the script in the first place. Originally the version set in the yaml file did not have any effect at all, and the default version that is set in the script itself was always effective (see https://github.com/gluster/centosci/blob/main/jobs/scripts/nightly-samba-builds/nightly-samba-builds.sh#L7). 

So this hunk should probably be a separate commit to enable handing in the variable from the yaml into the script.

**Review by obnoxxx — CHANGES_REQUESTED** [review]
I would prefer the commit message to not only say what is done but also why. Why are we doing it? Why are we doing it now and not earlier? What is changed behavior. ... And see my inline comment about splitting the commit into two.

File: jobs/samba-integration.yml (hunk 1 above)
Comment (anoopcs9) [reply jobs/samba-integration.yml:59]: Alright.

**Review by obnoxxx — APPROVED** [review]
lgtm

**Pull request merged**